TESTCHECKLIST.md

```diff
@@ -0,0 +1,38 @@
+# Module 9 Assessment Requirements Checklist and Test Plan
```

Comment:
Thanks for executing a test plan in this. It gives you immediate feedback on what we are going to focus on and it helps zero in on areas that need the attention.